### PR TITLE
SC Landing Tag

### PR DESCRIPTION
**Pull request by jmelvnsn**

### CODE REVIEW

**Comment by jmelvnsn** [comment]
@benghaziboy @MichaelButkovic @spra85 Please to bless this ship

File: bulbs/special_coverage/static/special_coverage/stylesheets/bulbs_sc.css [hunk 1]
```diff
@@ -0,0 +1,290 @@
+.image {
```
Comment (spra85) [inline bulbs/special_coverage/static/special_coverage/stylesheets/bulbs_sc.css]: Do we need to commit css?

File: bulbs/special_coverage/static/special_coverage/stylesheets/flexbox.scss [hunk 2]
```diff
@@ -0,0 +1,419 @@
+//////////////////////////
```
Comment (spra85) [inline bulbs/special_coverage/static/special_coverage/stylesheets/flexbox.scss]: Should we be shifting to have a single flexbox mixin pulled in somewhere more globally?  I feel like I've seen this kind of flexbox file replicated all over but maybe I'm crazy

File: bulbs/special_coverage/templates/special_coverage/bulbs_sc_landing.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  <div class="sc-label">Special Coverage</div>
+  <div class="billboard">
+    <h1>{{ special_coverage.name }}</h1>
+    {% if special_coverage.description %}
+      <p>{{ special_coverage.description }}</p>
+    {% endif %}
+    <div class="sponsor">
+      <div class="presented-by">
+        <span>Presented by</span>
+        {% if special_coverage.campaign.sponsor_logo %}
```
Comment (spra85) [inline bulbs/special_coverage/templates/special_coverage/bulbs_sc_landing.html]: We should coordinate the work on this with everything @kand, @benghaziboy and co have been doing all week as the `campaign` reference directly on special coverage is going away.

File: bulbs/special_coverage/static/special_coverage/stylesheets/bulbs_sc.css (hunk 1 above)
Comment (jmelvnsn) [inline bulbs/special_coverage/static/special_coverage/stylesheets/bulbs_sc.css]: At present, yes. We do not have a SASS compiler in the ClickHole's pipeline, and both the Onion and CH use LESS for their main stylesheets. I'll add this css to the pipeline of both sites for now.

File: bulbs/special_coverage/static/special_coverage/stylesheets/flexbox.scss (hunk 2 above)
Comment (jmelvnsn) [inline bulbs/special_coverage/static/special_coverage/stylesheets/flexbox.scss]: You have seen it more often than it should be replicated. However in this case we are pretty much starting from scratch here. We'll need to add a SASS compiler to django-bulbs as I was just using the SASS binary locally to make edits.

File: bulbs/special_coverage/templates/special_coverage/bulbs_sc_landing.html (hunk 3 above)
Comment (jmelvnsn) [inline bulbs/special_coverage/templates/special_coverage/bulbs_sc_landing.html]: I'll be sure to get their approval/make edits

**Comment by MichaelButkovic** [comment]
lgtm

**Pull request merged**